docs/usage/downloading-and-deploying.md

```diff
@@ -10,7 +10,7 @@ Pyodide packages, including the `pyodide.js` file, are available from the JsDeli
 
 | channel             | indexURL                                         | Comments                                                                                 | REPL                                               |
 | ------------------- | ------------------------------------------------ | ---------------------------------------------------------------------------------------- | -------------------------------------------------- |
-| Latest release      | `https://cdn.jsdelivr.net/pyodide/v0.18.0/full/` | Recommended, cached by the browser                                                       | [link](https://pyodide.org/en/stable/console.html) |
+| Latest release      | `https://cdn.jsdelivr.net/pyodide/v0.19.0/full/` | Recommended, cached by the browser                                                       | [link](https://pyodide.org/en/stable/console.html) |
```

Comment:
I think this still mentions the latest stable release which is fine. Once we bump the version we will also do it here, but we should do in a separate commit. Otherwise this link doesn't exist currently.
```suggestion
| Latest release      | `https://cdn.jsdelivr.net/pyodide/v0.18.0/full/` | Recommended, cached by the browser                                                       | [link](https://pyodide.org/en/stable/console.html) |
```